### ISSUE TITLE
Deprecated dependencies

### ISSUE THREAD

**Issue by stephenhand** [shalvah/twittersignin#12]
First of all, thanks for building this!

Unfortunately since it was last updated, there are some known security vulnerabilities in it's dependencies. It depends directly, and transitively via `twit` on `request`, which has been fully deprecated since 2020. Now the last version of `request` has a version of `json-schema` in it's tree that has a known vuln.

Since `request` and `twit` are the 2 most critical dependencies of `twittersignin` I appreciate it would take a major effort to rework it to remove them, but just wanted to make the maintainer(s) aware

**Comment by shalvah** [comment]
Yeah, it would be a lot of effort. I doubt this library is actually affected by the vulnerabilities, since it uses a very specific set of functionality. But it would definitely be comforting to end-users; however, I don't have the time. Best solution would probably be to get rid of those deps and just reimplement everything here. Some day (or someone's PR...).
